inc/notificationtarget.class.php

```diff
@@ -1536,6 +1536,21 @@ function getUserByField($field, $search_in_object = false) {
    }
 
    /**
+    * Add new recipient with lang to current recipients array
+    *
+    * @param array $data Data (users_id, lang[, field used for notification])
+    *
+    * @deprecated Use NotificationTarget::addToRecipientsList()
+    *
```

Comment:
I've added the version. Since the phpdoc specs talks about `@deprecated [<version>] [<description>]` (see https://phpdoc.org/docs/latest/references/phpdoc/tags/deprecated.html); I've also updated the few existing to fit.